package.json

```diff
@@ -70,6 +71,6 @@
     "terser": "^5.10.0",
     "ts-jest": "^27.1.2",
     "ts-node": "^10.4.0",
-    "typescript": "^4.5.4"
+    "typescript": "^5.5.4"
```

Comment:
I avoided upgrading the rest of the dependencies because there were issues in your shared config